commands/login/index.js

```diff
@@ -37,6 +56,7 @@ program
     } catch (e) {
       return logError(e);
     }
+
     try {
       const wlInfo = await api.request({ method: 'get', url: '/whitelabels/domain' });
```

Comment:
should you move this line into the retry-able function? that way if you typo your access token you get asked again instead of it exiting?